extracted-files/tags/record-SOUR

```diff
@@ -4,7 +4,7 @@ lang: en-US
 
 type: structure
 
-uri: https://gedcom.io/terms/v7/record-SOUR
+uri: https://gedcom.io/terms/v7.1/record-SOUR
```

Comment:
I think our rule should be that you can't change the URI in an existing non-prerelease YAML file.  Hence instead of modifying record-SOUR, this PR should create a new record-SOUR-v7.1 or some similar filename.